api/base/settings/__init__.py

```diff
@@ -11,19 +11,27 @@
 import itertools
 
 from .defaults import *  # noqa
+from website import settings
 
 try:
     from .local import *  # noqa
 except ImportError as error:
     warnings.warn('No api/base/settings/local.py settings file found. Did you remember to '
                   'copy local-dist.py to local.py?', ImportWarning)
 
+if not settings.DEV_MODE:
+    if not local.BYPASS_THROTTLE_TOKEN:
+        warnings.warn('Local throttle token should be set.')
+    else:
+        assert local.BYPASS_THROTTLE_TOKEN != defaults.BYPASS_THROTTLE_TOKEN, 'Local throttle token should be changed from the default value.'
```

Comment:
You could probably just add `'BYPASS_THROTTLE_TOKEN'` to the tuple in the `for` loop below.